flow/util/genMetrics.py

```diff
@@ -434,9 +439,14 @@ def extract_metrics(cwd, platform, design, flow_variant, output, hier_json):
         for design_it in os.scandir(platform_it.path):
             if not design_it.is_dir():
                 continue
+            des = design_it.name
+            if not (all_designs or des in designs):
+                continue
             for variant in flow_variants:
-                des = design_it.name
-                print(plt, des, variant)
+                ddd = os.path.join(cwd, 'reports', plt, des, variant)
+                if not os.path.isdir(ddd):
```

Comment:
What is `ddd`?

Reply:
that was an oversight  I thought I had renamed the variable -- fixed now